src/redux/rockets/reducer.js

```diff
@@ -35,4 +42,11 @@ export const reserveRocket = (id) => (dispatch) => {
   });
 };
 
+export const cancelRocket = (id) => (dispatch) => {
+  dispatch({
+    type: CANCEL_ROCKET,
+    payload: id,
+  });
+};
```

Comment:
Well, when almost the entire concept of the `reserveRocket` and the `cancelRocket` is the same, what do you think to remove the one which smells duplicated?